ecs-cli/modules/cli/compose/adapter/containerconfig.go

```diff
@@ -32,6 +32,7 @@ type ContainerConfig struct {
 	PseudoTerminal        bool
 	ReadOnly              bool
 	ShmSize               int64
+	StopTimeout           *int64
```

Comment:
Why a pointer? it seems like our other fields just use the concrete type.

Reply:
Ah nvm I see, it's because we want `nil` to be an actually valid value 👍. 

Do you think we should add a comment here explaining that?